lab-adan/test/movie-router-test.js

```diff
@@ -0,0 +1,83 @@
+require('dotenv').config({ path: `${__dirname}/../.test.env` });
+const superagent = require('superagent');
+const expect = require('expect');
+const server = require('../lib/server.js');
+
+const API_URL = `http://localhost:${process.env.PORT}`;
+let tempGame;
+
```

Comment:
Keep going on these tests. Let me know if you have any questions about setting them up.